src/map/script.c

```diff
@@ -26957,13 +27073,14 @@ static void script_parse_builtin(void)
 		BUILDIN_DEF(changebase,"i?"),
 		BUILDIN_DEF(changesex,""),
 		BUILDIN_DEF(changecharsex,""), // [4144]
-		BUILDIN_DEF(waitingroom,"si?????"),
+		BUILDIN_DEF(waitingroom,"si??????"),
```

Comment:
```suggestion
		BUILDIN_DEF(waitingroom, "si??????"),
```
Same for the other touched `BUILDIN_DEF()`.

Reply:
updated